package.json

```diff
@@ -18,12 +18,6 @@
     "@open-wc/rollup-plugin-html": "^0.3.4",
     "@rollup/plugin-node-resolve": "^7.1.1",
     "babel-plugin-bundled-import-meta": "^0.3.2",
-    "es-dev-server": "^1.45.2",
```

Comment:
Dont remove the dev server please, we need this

Reply:
Ok Pascal, I have added the es-dev-server extension back in the package.json file and have pushed it.